File: backend/src/SGPI/Core/Entities/Estoque.cs
namespace SGPI.Core.Entities
{
    public class Estoque
    {
        public int Id { get; set; }
        public int ItemCatalogoId { get; set; }
        public ItemCatalogo? ItemCatalogo { get; set; }
        public int EstoqueMinimo { get; set; }
        public int EstoqueMaximo { get; set; }
        public int QuantidadeEmEstoque { get; set; }
    }
}

File: frontend/src/app/interfaces/estoque.ts
import { ItemCatalogo } from './item-catalogo';

export interface Estoque {
  id: number;
  itemCatalogoId: number;
  itemCatalogo?: ItemCatalogo;
  estoqueMinimo: number;
  estoqueMaximo: number;
  quantidadeEmEstoque: number;
}
